atomix/cluster/src/main/java/io/atomix/cluster/protocol/HeartbeatMembershipProtocol.java

```diff
@@ -358,11 +358,6 @@ public String name() {
       return NAME;
```

Comment:
@npepinpe I assume we will stay with SWIM? So this might be also something we can remove?